velero-plugin-for-microsoft-azure/volume_snapshotter.go

```diff
@@ -145,6 +148,28 @@ func (b *VolumeSnapshotter) Init(config map[string]string) error {
 		snapshotsIncremental = nil
 	}
 
+	// if config["snapsSkuConfigKey"] is empty, default to nil; otherwise, convert it
+	var snapshotsSku *disk.SnapshotSku
+    if val := config[snapsSkuConfigKey]; val != "" {
+        var snapshotsSkuName disk.SnapshotStorageAccountTypes
+        var found bool
+        for _, name := range disk.PossibleSnapshotStorageAccountTypesValues() {
```

Comment:
Looking at the possible [values and their descriptions](https://github.com/Azure/azure-sdk-for-go/blob/v42.0.0/services/compute/mgmt/2019-07-01/compute/models.go#L832-L844), it seems like different SKU types apply to different storage types, e.g. `Premium_LRS` is for Premium SSDs, `Standard_LRS` is for standard HDDs. If a user sets the SKU but has a mix of storage types, what is the behaviour? Does it fall back to an appropriate SKU for that storage type?